Profiles/cio-Symptom.xml

```diff
@@ -0,0 +1,153 @@
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="cio-dataset-200-beta2-20231214" />
+    <uri value="https://decor.nictiz.nl/pub/cio/cio-html-20231214T100742/ds-2.16.840.1.113883.2.4.3.11.60.26.1.2-2020-04-02T000000.html" />
+    <name value="ART-DECOR Dataset Contraindications and hypersensitivities 2.0.0-beta.2 20231214" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="Symptom" />
+      <definition value="A phenomenon with which a condition in the patient presents itself." />
+      <comment value="This also includes a complaint or a finding, but not additional research, such as laboratory tests, function tests and imaging studies." />
+      <alias value="Symptoom" />
+      <mapping>
+        <identity value="cio-dataset-200-beta2-20231214" />
+        <map value="cio-dataset-200-beta2-890" />
+        <comment value="Symptom" />
+      </mapping>
+    </element>
+    <element id="Observation.extension">
+      <path value="Observation.extension" />
+      <min value="1" />
+    </element>
+    <element id="Observation.extension:medicationHypersensitivityIdentifier">
+      <path value="Observation.extension" />
+      <sliceName value="medicationHypersensitivityIdentifier" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationHypersensitivityIdentifier" />
+      </type>
+      <isModifier value="false" />
+    </element>
+    <element id="Observation.extension:medicationHypersensitivityIdentifier.url">
+      <path value="Observation.extension.url" />
+      <fixedUri value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationHypersensitivityIdentifier" />
+    </element>
+    <element id="Observation.extension:relationCondition">
```

Comment:
@LuudSlagter  I haven't thought of mapping RelationCondition via `Condition.evidence.detail` but I do support that idea for Symptom. As the zib page describes: _De Aandoening of gesteldheid presenteert zich met symptomen_. So linking the Symptom through the `.evidence` element seems logical. 

However, I'm not a fan of mapping Reaction and Hypersensitivity through `.evidence.detail`. I don't think it is correct to say that Condition has become a more central element. A SurveillanceDecision is primarily linked to a Reaction or a Hypersensitivity via DecisionBasis. A Condition used more as context for the Reaction and Hypersensitivity and not the other way around (the Reaction and Hypersensitivity as "evidence" for the Condition. 

Based on this, I would opt for a hybrid situation.

Reply:
@LuudSlagter I have analyzed this a bit further and I don't think the current Symptom zib en dataset structure is fit for the mapping of RelationCondition through `Condition.evidence.detail`. The name of the dataset element is "Relation**Condition**" and the definition is "_The condition of which the symptom is a manifestation_" which does not make sense to map inside of a Condition profile. Changing the name and description would be confusing and does not make sense. I wanted to suggest to IA's to move the relation between Condition and Symptom to go from Condition to Symptom instead of Symptom to Condition. However, the zib page states that there is a link from Symptom to Condition and not the other way around. I assume we cannot simply change the direction without any consequences. 
I will discuss this point with IA's.

Reply:
@LuudSlagter okay so I thought about this a little bit and checked it with Alexander. In conclusion I chose to omit the mapping of RelationCondition and added a relation between Condition and Symptom through `.evidence.detail`. I will also add some supporting comments to Symptom and Condition profile.

Reply:
See https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4#References_that_are_reversed_in_FHIR for guidance on how to model such a concept.